lbcapi/api.py

```diff
@@ -79,7 +91,12 @@ def call(self, method, url, params={}, stream=False, files=None):
             # Loop, so retrying is possible if nonce fails
             while True:
 
-                nonce = long(time.time() * 1000)
+                if sys.version_info >= (3, 0):
+                    # There is no "long" type in py3
+                    nonce = int(time.time()) * 10**3
```

Comment:
Here is a problem. If you round time.time() before multiplying it, then there will be a nonce collision with all calls that are made during same second. Also 10**3 is unnecessary complexity, better just use 1000.

Anyway, good point that Python 3 does not have long(). int() works fine in Python 2 too, so best solution would be to just replace "long" with "int", so the whole nonce calculation would be:

```
nonce = int(time.time() * 1000)
```